src/migrations/20190918144932_create_userStats_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS userStats(
+    id INTEGER auto_increment primary key,
+    user_id INTEGER not null,
+    created_challenges INTEGER,
+    solved_challenges INTEGER,
+    current_points INTEGER,
+    created_at DATETIME,
+	updated_at DATETIME
+)ENGINE=InnoDB;
```

Comment:
I believe ENGINE=InnoDB; is not needed. Also, a newline should be added after ;.